### ISSUE TITLE
Couldn't fetch RxKeyboard as dependency thru Carthage (Swift 5)

### ISSUE THREAD

**Issue by SergeMaslyakov** [RxSwiftCommunity/RxKeyboard#89]
Hi guys!

I use Carthage. And when I try to fetch dependencies I get this error:
```
*** Downloading RxKeyboard.framework binary at "0.9.2"
***  Skipped installing RxKeyboard.framework binary due to the error:
	"Incompatible Swift version - framework was built with 4.2.1 (swiftlang-1000.11.42 clang-1000.11.45.1) and the local version is 5.0 (swiftlang-1001.0.69.5 clang-1001.0.46.3)."

    Falling back to building from the source
``` 
and then below
```
*** Skipped building RxKeyboard due to the error:
Dependency "RxKeyboard" has no shared framework schemes for any of the platforms: iOS

If you believe this to be an error, please file an issue with the maintainers at https://github.com/RxSwiftCommunity/RxKeyboard/issues/new
```

Could you release a new version with a binary compatibly with Swift5?
Right now I just added the sources directly in my project and everything works. But..

Thanks!

**Comment by freak4pc** [comment]
This is ongoing. See #88.

**Comment by kikisasha** [comment]
The issue still happening.

Did anyone manage to pass this related error ?
Dependency "RxKeyboard" has no shared framework schemes

If you believe this to be an error, please file an issue with the maintainers at https://github.com/RxSwiftCommunity/RxKeyboard/issues/new

**Comment by kikisasha** [comment]
Hey @SergeMaslyakov 

Do you happen to have an alternative library to fix this issue for temporary use to wait for the fix release?

**Comment by SergeMaslyakov** [comment]
Hey @kikisasha 

I have forked repo and have addded fix for Carthage.
You could use it if you need.

- repo
[https://github.com/BlackBrickOrg/RxKeyboard](https://github.com/BlackBrickOrg/RxKeyboard)

- Cartfile
`github "BlackBrickOrg/RxKeyboard" "0.9.5"`

**Comment by kikisasha** [comment]
@SergeMaslyakov thanks for the help,

But the repo you gave me seems to be supporting above iOS 10.0
Am I correct?
Is it possible to make a branch that support minimum iOS 9.0 though?

**Comment by SergeMaslyakov** [comment]
@kikisasha Yes, it is. I have changed it for my purposes. You just can revert it in xcode proj settings.

**Comment by kikisasha** [comment]
@SergeMaslyakov I did download the repo but some how error pops up saying "No such module "RxRelay"

Any suggestions?

**Comment by SergeMaslyakov** [comment]
@kikisasha 

1) Check `RxSwift` version which you have installed (in `Cartfile.resolved`). It should be >= 5.0.1.
2) Make sure that you added all built Rx-related frameworks in carthage copy phase in your project
<img width="1051" alt="Screenshot 2020-03-18 at 11 27 31" src="https://user-images.githubusercontent.com/29630110/76940516-78c5b100-690b-11ea-85c5-9e01abe7c9cf.png">
